default.py

```diff
@@ -290,13 +291,15 @@ def listLive(url):
     xbmcplugin.endOfDirectory(pluginhandle)
 
 
-def search():
+def search(url):
     keyboard = xbmc.Keyboard('', translation(30002))
     keyboard.doModal()
     if keyboard.isConfirmed() and keyboard.getText():
         search_string = urllib.parse.quote_plus(keyboard.getText())
-        listVideos(urlMain + "/videos?fields=description,duration,id,owner.username,taken_time,thumbnail_large_url,title,views_total&search=" + search_string + "&sort=relevance&limit=" + itemsPerPage + "&family_filter=" + familyFilter + "&localization=" + language + "&page=1")
+        url2 = url.replace("&search=", "&search=" + str(search_string))
```

Comment:
Thanks for the PR, Did you test this line in Kodi19 nightly,
May be better to use it this way
```suggestion
        url2 = url.replace("&search=", "&search=".format(search_string))
```

Reply:
`url2 = url.replace("&search=", "&search=" + str(search_string))` ok 👍 
`url2 = url.replace("&search=", "&search={}".format(search_string))` ok 👍 
`url2 = url.replace("&search=", "&search=".format(search_string))` don't works

Reply:
yep, missed the `{}`
```url2 = url.replace("&search=", "&search={}".format(search_string))``` is correct